### PR TITLE
Release/0.9.0 revert styled components

### PR DESCRIPTION
**Pull request by ericmarcos**
Picking up from #281 but removing styled-components commits.
We should investigate further what's going on, but some styles defined with styled-components don't show up in production build (message, buttons) but some others do (header). As it's not critical, I've switched to inline styles to make sure it works so we can move forward with the release.

### CODE REVIEW

File: packages/botonic-react/src/webchat/webchat.jsx [hunk 1]
```diff
@@ -371,115 +528,30 @@ export const Webchat = forwardRef((props, ref) => {
...
-                style={{
-                  ...props.theme.webviewStyle
-                }}
-                webview={webchatState.webview}
-              />
-            </RequestContext.Provider>
+          ) : (
+            <>
+              {webchatMessageList()}
+              {webchatState.replies &&
```
Comment (guinii) [inline packages/botonic-react/src/webchat/webchat.jsx]: The replies, is going to be rendered fixed at the end of the message list. It should be as a new message that goes away when scrolling

File: packages/botonic-react/src/webchat/webchat.jsx [hunk 2]
```diff
@@ -320,13 +375,115 @@ export const Webchat = forwardRef((props, ref) => {
         ...webchatState.theme.triggerButtonStyle
       }}
     >
-      <img
+      {triggerImage && (
+        <img
+          style={{
+            height: 50
```
Comment (guinii) [inline packages/botonic-react/src/webchat/webchat.jsx]: Maybe put some style option in order to define the size of the triggerImage. I don't know if it's better this option or force the user to create a CustomTriggerButton

File: packages/botonic-react/src/components/persistentMenu.jsx [hunk 3]
```diff
@@ -4,6 +4,7 @@ export const PersistentMenu = props => {
     <div
       style={{
         position: 'absolute',
+        zIndex: 1,
```
[the remark this answers is not in the document]
Comment (ericmarcos) [reply packages/botonic-react/src/components/persistentMenu.jsx]: z-index is relative to the container. In this case we want the persistent menu to be in front of the other elements of our webchat.
If someone has visibility problems with the webchat on their website, the only solution is that they setup the zindex of the #root div manually.

File: packages/botonic-react/src/webchat/header.jsx [hunk 4]
```diff
@@ -1,69 +1,76 @@
 import React, { useContext } from 'react'
 import { WebchatContext } from '../contexts'
+import { staticAsset } from '../utils'
 import styled from 'styled-components'
```
[the remark this answers is not in the document]
Comment (ericmarcos) [reply packages/botonic-react/src/webchat/header.jsx]: yes.. they work ok in header elements. Not sure why.
I'll leave it like this as it's working

File: packages/botonic-react/src/webchat/webchat.jsx (hunk 1 above)
Comment (ericmarcos) [reply packages/botonic-react/src/webchat/webchat.jsx]: I'll leave it like this for now. I'd like to add a Messenger-like behaviour: replies are fixed at the end but they disappear if you scroll up more than X%

**Pull request merged**